src/scim2_filter_parser/transpilers/django_q_object.py

```diff
@@ -236,5 +236,13 @@ def lookup_op(node_value, comp_value):
             #  "<field> iexact true/false" to "UPPER(field::text) = UPPER(true/false), which fails.
             #  UPPER requires a string.
             return "exact"
+        if comp_value == "" and op == "iexact":
+            # In Oracle iexact + empty string will never evaluate to true. I.e., TRIM(' ') != '' and
```

Comment:
nit: let's indent all the comments evenly.

Reply:
I hope I got the suggestion right 👌